server.js

```diff
@@ -1,9 +1,115 @@
 const express = require('express');
+const db = require('./data/db');
 
+const port = 5555;
```

Comment:
Nice use of a `const` to determine port number. Can be changed easier in one place if needed. 👍